### PR TITLE
chore(cli): fix regression tests execution wrapping

### PR DESCRIPTION
**Pull request by iliapolo**
### Commit Message

Follow up to https://github.com/aws/aws-cdk/pull/7180.

The exclusion mechanism itself lives inside [`test.sh`](https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/test/integ/cli/test.sh#L38), which is part the integ directory that is being replaced during the execution of the regression tests. This means that we have to use the current version of `test.sh`. 

Same goes for [`common.bash`](https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/test/integ/cli/common.bash#L74), which contains a [fix](https://github.com/aws/aws-cdk/pull/7060/files#r399851571) that we need. 

Basically, we use the common and wrapping code of the current version, with tests from the previous version. 

This is not ideal, but can work for now.

### End Commit Message

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk/test/integ/test-cli-regression-against-current-code.sh [hunk 1]
```diff
@@ -69,6 +69,10 @@ integ_under_test=${integdir}/cli-backwards-tests-${sanitized_version}
 rm -rf ${integ_under_test}
 echo "Copying integration tests of version ${VERSION_UNDER_TEST} to ${integ_under_test} (dont worry, its gitignored)"
 cp -r ${temp_dir}/packages/aws-cdk/test/integ/cli ${integ_under_test}
+echo "Copying test runner to integration tests of version ${VERSION_UNDER_TEST}"
```
Comment (eladb) [inline packages/aws-cdk/test/integ/test-cli-regression-against-current-code.sh:72]: yak
